pymc3/tests/test_distributions.py

```diff
@@ -219,6 +220,13 @@ def build_model(distfam, valuedomain, vardomains, extra_args=None):
     return m
 
 
+def laplace_asymmetric_logpdf(value, kappa, b=None):
+    kapinv = 1 / kappa
+    lPx = value * np.where(value >= 0, -kappa, kapinv)
+    lPx -= np.log(kappa + kapinv)
```

Comment:
We need to use scale parameter `b` as well.
```suggestion
    lPx = value * b * np.where(value >= 0, -kappa, kapinv)
    lPx += np.log(b / (kappa + kapinv))
```

Reply:
This is the code that I took from SciPy repo. Since they are using only two parameters `kappa` and `value`, I did use the same.
In case, If I am suppose to edit these, please do let me know.

Reply:
I am looking at https://scipy.github.io/devdocs/generated/scipy.stats.laplace_asymmetric.html. And it indeed uses loc, scale and kappa parameters. 
On the other hand, `b` parameter is in this distribution. So it should be tested in logpdf.